test/integration/install/test_install.py

```diff
@@ -160,3 +160,44 @@ def test_install_tarantool_in_docker(tt_cmd, tmpdir):
     assert out == "GLIBC_2.27"
 
     assert os.path.exists(os.path.join(tmpdir, "my_inc", "include", "tarantool"))
+
+
+def test_install_tarantool_dev(tt_cmd, tmpdir):
+    build_dirs = [
+        os.path.join(tmpdir, "build1"),
+        os.path.join(tmpdir, "build2")
+    ]
+    build_rel_paths = ['src', 'tarantool/src']
+
+    # Create imitations of tarantool binaries.
```

Comment:
Add checks for include dir.
Check that symlink is updated if already exists for binary, for include dir. Check the scenario if `tarantool` exists in bin dir and is not a symlink. The same is for include dir.
Add test for `--include-dir` option and default include dir search. `--include-dir` pointing to non-existing dir, to the file scenarios.
Add test for uninstall. Cover different scenarios additionally: broken symlink for binary, for include dir, tarantool symlink pointing to binary in `bin` (should not be removed).